scipy/stats/tests/test_sampling.py

```diff
@@ -312,6 +312,100 @@ def check_discr_samples(rng, pv, mv_ex):
     assert pval > 0.1
 
 
+@pytest.mark.parametrize('method', ["NumericalInverseHermite",
+                                    "NumericalInversePolynomial"])
+class TestQRVS:
+    def test_input_validation(self, method):
+        match = "`qmc_engine` must be an instance of..."
+        with pytest.raises(ValueError, match=match):
+            Method = getattr(stats.sampling, method)
+            gen = Method(StandardNormal())
+            gen.qrvs(qmc_engine=0)
+
+        if NumpyVersion(np.__version__) >= '1.18.0':
```

Comment:
We can remove this. The oldest support numpy is `1.18.5` in SciPy 1.9.0.

Reply:
Good catch, I missed it as it was a move!